tailscale/README.md

```diff
@@ -0,0 +1,66 @@
+# Agent Check: Tailscale
+
+## Overview
+
+Tailscale (https://tailscale.com/) is a peer-to-peer VPN service that simplifies and secures network connectivity.
+
+With this integration, you can:
+
+1. Control your Tailscale data retention.
+2. Build custom widgets and dashboards.
+3. Setup your own or leverage out of the box detection rules
+4. Cross-reference Tailscale events with the data from other services in your stack.
+
+This integration streams two log types from Tailscale:
+
+[Configuration Audit Logs](https://tailscale.com/kb/1203/audit-logging/):
```

Comment:
This type of link is acceptable also. The documentation team prefers the numbered ones because they are easier for our build and translation processes to parse.